### ISSUE TITLE
Короткие ссылки на значимые локации или населённые пункты

### ISSUE THREAD

**Issue by evgenykatyshev** [heritagemap/frontend-pwa#202]
Было бы прикольно иметь возможность добавлять короткие ссылки на какие-то значимые населённые пункты или локации. Например, сделать ссылку heritagemap.ru/nizhny-novgorod, которая редиректила бы на координаты Нижнего с подходящим зумом.

Имею в виду возможность прописать это где-то в коде, а не возможность для любого пользователя.

**Comment by Julia-Lavrova** [comment]
Не совсем поняла. Тут нужен конкретный список захардкоженный? Если так, то нужен просто список какие добавить

**Comment by evgenykatyshev** [comment]
Да, захардкорженный, желательно отдельным файликом. Можно совсем просто в виде пар ссылок:

https://heritagemap.ru/lat/56.301011/lon/43.995229 heritagemap.ru/nizhny-novgorod
https://heritagemap.ru/lat/55.744654/lon/37.624991 heritagemap.ru/moscow

Но лучше что-то типа такого (lat,lon,zoom название места):

56.301011,43.995229,12 nizhny-novgorod
55.744654,37.624991,12 moscow

Координаты или ссылки с координатами лучше вначале, т. к. у них одинаковая ширина будет — удобнее читать. Сделай для двух городов пока (координаты реальные), я потом добавлю остальные.

**Comment by Julia-Lavrova** [comment]
👌

пн, 25 янв. 2021 г. в 18:55, Evgeny Katyshev <notifications@github.com>:

**Comment by Julia-Lavrova** [comment]
@evgenykatyshev Проверяй, если будешь еще добавлять, то замечу, что редирект должен быть выше объявления роута `/lat/:lat/lon/:lon/:id`  иначе по нему пройдет

**Comment by evgenykatyshev** [comment]
В браузере работает, а если ссылка открывается в приложении или во встроенном в другое приложении браузере (по ссылке из Телеграма, например), то не срабатывает редирект.
